lnwallet/channel.go

```diff
@@ -3513,7 +3537,11 @@ func (lc *LightningChannel) ProcessChanSyncMsg(
 		commitPoint = lc.channelState.RemoteNextRevocation
 	}
 
-	if commitPoint != nil &&
+	// We only need to check the unrevoked commitment point value if this
+	// isn't a tweakless commit, as otherwise, it'll be zero.
```

Comment:
we're still setting it in `ChanSyncMsg()`

Reply:
Ah yeah you're right, will wait until we fix the issue in the current spec draft re _removing_ it all together. Also FWIW, there's no harm in still setting it (other than the extra CPU cycles to compute the point from the secret).